lib/matplotlib/colorbar.py

```diff
@@ -1361,10 +1376,12 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
         anchor=anchor,
         panchor=panchor,
         fraction=fraction,
-        aspect=aspect,
+        aspect=aspect0,
         pad=pad)
     # and we need to set the aspect ratio by hand...
-    cax.set_aspect(aspect, anchor=anchor, adjustable='box')
+    cax.set_anchor(anchor)
+    cax.set_box_aspect(aspect)
+    cax.set_aspect('auto')
```

Comment:
Maybe can replace this (and other aspect setters) with `self._set_aspect` so it gets coverage? In fact, there are no callers of it, now that it's not public.

Reply:
Unfortunately we can't because the Colorbar object has not been made yet.  This is just the routine that makes the empty axes.  I guess we could set the aspect in the actual call to colorbar, but that would be a public API change to not allow it to have an effect here.